### DISCUSSION TITLE
Making a new DDS website, or integrating it into the current ecosystem (or both)

### DISCUSSION THREAD

**Discussion by matthewmccarthy11** [life-itself/community#861] [Research]
One of the deliverables of the DDS project that we have been discussing is to make a website to compile the resources, and it has been suggested that the website looks something like :https://metacrisis.org/META-CRISIS/00.+%F0%9F%91%8B+About/How+do+I+add%2C+edit%2C+or+remove+pages%3F

However, Life Itself already has a substantial 'ecosystem map' which compiles a number of different organizations, including certain metrics, and the question worth considering here is whether or not we want to make our own new website as it relates to DDS, or integrate it into the Life Itself ecosystem. Of course, it might be possible in the long run to somehow do both, but this thread is set up to consider and discuss the potential pros and cons of each different setup, at least at this stage. For clarity's sake, **option 1** refers to a new DDS website, **option 2** refers to integrating DDS resources into the current and existing Life itself map. Here are just  few pros and cons of each

**option 1**

- Allows for us to visualize more clearly the connections between organizations, language, ecologies of practice, etc. 
- Can exist 'on its own' and might encourage others to contribute more, since it is not directly connected to one organization (although I'm not entirely sure if this is true) 
- Might be difficult to integrate it into the existing ecosystem map as an entire 'section', so starting from scratch makes it (maybe) simpler

**option 2**

- There is already so much information, might as well build off what is already there
- How, though, can it be meaningfully integrated? 
- There is a sense in which building off of one thing and making more of a 'mega resource' instead of many smaller ones might make more sense- and this may or may not also be worth considering as it relates to Cohere+

**Comment by rufuspollock** [comment]
Good summary with options 🙏

I think we go for integrating this into ecosystem site ... and i think that means we reboot ecosystem site as a separate url at https://ecosystem.lifeitself.org (and move it out of lifeitself.org/ecosystem). /cc @catherinet1

**Reply by BoazBF** [reply]
Thanks @rufuspollock , is it possible for you to sync with @matthewmccarthy11 to see if a prototype is possible to show for the Oct 30 Coalescing call? Or is this too complicated for now, and would only be something we mention as an option for further work to explore in the coalescing cal?

**Reply by rufuspollock** [reply]
@BoazBF i can sync. I would reckon on including @catherinet1 has a lot more background knowledge here re the ecosystem site.

And yes if we get moving we can have a site for the October 30th call. I think it requires a 1-2h planning call asap.

**Reply by matthewmccarthy11** [reply]
I can sync tomorrow anytime from 12:00 CET onwards, other than 18:30-19:30
